src/libsync/owncloudpropagator.cpp

```diff
@@ -766,8 +766,6 @@ bool OwncloudPropagator::createConflict(const SyncFileItemPtr &item,
         return false;
     }
 
-    emit touchedFile(fn);
-    emit touchedFile(conflictFilePath);
```

Comment:
As we no longer announce the creation of the conflict file, we can no longer misuse the signal for the unit test.